Doppel : Use System Proxy Settings, Organization Code and Retry logic updates (#49)
metron-labs/content: Packs/Doppel/ReleaseNotes/1_0_4.md

Reviewer: Also mention about the Screenshot URL field

Author: I have updated the release notes, thanks!

#### Integrations

##### Doppel
- Updated the Docker image to: *demisto/python3:3.12.11.4819260*.

> - Added support for **Organization Code (optional)**, **Use System Proxy** and **Trust Any Certificate (insecure)** parameters to allow users to use organization code provided by Doppel and use proxy with their cortex engine. 

Reviewer: ```suggestion
- Added support for **Organization Code**, **Proxy**, and **SSL flag** parameters to allow users to use the organization code provided by Doppel and use a proxy with their Cortex engine. 
```

#### Integrations

##### Doppel
- Updated the Docker image to: *demisto/python3:3.12.11.4819260*.

- Added support for **Organization Code**, **Proxy**, and **SSL flag** parameters to allow users to use the organization code provided by Doppel and use a proxy with their Cortex engine. 

> - Added new Incident Field as Screenshot URL.

Reviewer: Isn't it Doppel Screenshot URL?

Author: Fixed, thanks!
